drupal/composer.json

```diff
@@ -126,7 +126,7 @@
                 "Get same results on different request": "https://www.drupal.org/files/issues/2019-07-18/change_request_type-63049395-09.patch"
             },
             "drupal/decoupled_router": {
-                "Unable to resolve path on node in other language than default": "https://www.drupal.org/files/issues/2022-12-01/decouple_router-3111456-resolve-language-issue-58--get-translation.patch"
+                "Unable to resolve path on node in other language than default": "https://www.drupal.org/files/issues/2023-04-27/decouple_router-3111456-resolve-language-issue-64--get-translation.patch"
```

Comment:
https://www.drupal.org/project/decoupled_router/issues/3111456#comment-15027425 This version of the patch contains the fix we needed.